.travis.yml

```diff
@@ -23,6 +23,8 @@ before_script:
 script:
     # flake8 the diff
     - "git diff HEAD^ '*.py' | flake8 --diff --config flake8.cfg"
+    # add secret key to django settings.py
+    - echo "SECRET_KEY= '$(cat /dev/urandom | tr -dc 'a-z0-9!@#$%^&*(-_=+)' | head -c 50)'" >> platform/pulpcore/app/settings.py
```

Comment:
I tried making a /etc/pulp/server.yaml file on travis, but it threw a 'exit code 1' error that I assumed to be due to lack of permission. This was the next best thing that worked :)